frontend/src/pull-card/refresh.tsx

```diff
@@ -0,0 +1,51 @@
+import { Pull } from '../pull';
+import { refreshPull } from '../backend/pull-socket';
+import { Box, chakra } from "@chakra-ui/react"
+import { useState } from "react"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faRefresh } from '@fortawesome/free-solid-svg-icons'
+
+const RefreshContainer = chakra(Box, {
+   baseStyle: {
+      lineHeight: 0.9,
+      visibility: "hidden",
+      position: "absolute",
+      cursor: 'pointer',
+      bottom: '3px',
+      right: '3px',
+      borderRadius: '4px',
+      borderWidth: '1px',
+      borderColor: 'var(--refresh)',
+      padding: "5px",
+      background: "white",
+      color: "var(--refresh)",
+      "&.refreshing": {
+         visibility: "visible",
+      },
+      "&:hover": {
+         color: "var(--refresh-hover)",
+         borderColor: "var(--refresh-hover)",
+      },
+      "&:active": {
+         color: "white",
```

Comment:
When it comes to setting the colors, do we usually want to opt for colors defined in the `*.less` files?

Reply:
Thanks fro prompting, I looked into it this. We could make a new `--refresh-active-color: @white` but It's used *just* during the click operation (i.e. sub-1s) and it's gonna be white in both cases, I'd rather not bloat the theme with unimportant distinctions that don't need to change between dark/light.